tutorindigo/templates/indigo/lms/static/sass/courseware/_discover.scss

```diff
@@ -136,6 +139,104 @@
         clear: both;
         content: "";
     }
+    .content-area {
+        display: flex; // Align the sidebar and courses horizontally
+        flex-direction: row;
```

Comment:
if you want to make it responsive

```scss
 .content-area {
// ... all the styles
 flex-direction: column;
@include media-breakpoint-down(sm)){
    flex-direction: row;
  }
}
```

Reply:
I tried your suggestion, but it worked when I inverted the properties.
Thanks!

Commit: https://github.com/eduNEXT/tutor-indigo/pull/1/commits/8651a45bd3f4a1264e69152edc5d88c8c7935f84

Iĺl merge it. If we need to improve this, it'll be in another PR.